Joke Machine/Controller/ViewController.swift

```diff
@@ -38,7 +36,7 @@ class ViewController: UIViewController {
         }.resume()
```

Comment:
print("something is wrong") serine alert basabilirsin kullanıcıyı doğru yönlendirmek açısından